### PR TITLE
Update Maestro subscriptions to trigger corefx release/uwp6.1 updates

### PR DESCRIPTION
**Pull request by zamont**
Update Maestro subscriptions to trigger corefx release/uwp6.1 updates on corefx & coreclr release/uwp6.1 updates

### CODE REVIEW

**Comment by zamont** [comment]
cc: @dagood @nattress

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -452,6 +452,32 @@
...
+    // Update dependencies in CoreFX release/uwp6.1
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/release/uwp6.1/Latest.txt",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/coreclr/release/uwp6.1/Latest.txt"
+      ],
+      "action": "corefx-general",
+      "delay": "00:10:00",
+      "actionArguments": {
+        "vsoSourceBranch": "release/uwp6.1",
```
Comment (dagood) [inline Maestro/subscriptions.json]: Doesn't look like it's set up yet, still has `<Uwp6ReleaseDependencyBranch>release/uwp6.0</Uwp6ReleaseDependencyBranch>` (https://github.com/dotnet/corefx/blob/98648ec5143a194fa1326dd3f501272a63357b75/dependencies.props#L50)

Comment (dagood) [reply Maestro/subscriptions.json]: Oh, I see https://github.com/dotnet/corefx/pull/28617 now. :smile:

Comment (dagood) [reply Maestro/subscriptions.json]: And actually, no need to wait. The worst that happens is that `release/uwp6.1` gets a weird auto-PR that's fixed as soon the CoreFX PR is merged and a real uwp6.1 build comes around. Merging.

**Pull request merged**